examples/to-do/index.es

```diff
@@ -0,0 +1,78 @@
+Element `to-do`
+
```

Comment:
Hmm well best part is it's just whitespace so we can author how we please. :-) @brandondees